### PR TITLE
Rename baci to 4C

### PR DESCRIPTION
**Pull request by isteinbrecher**
#64

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -17,17 +17,17 @@ env:
   # Indicated the testing script, that it is performed on GitHub. In this
   # case all test have to pass.
   TESTING_GITHUB: 1
-  # Some test require baci-release to check if the created input file
-  # works with baci.
-  BACI_BUILD: /home_local/github-runner/testing_lib/baci_master/release
-  BACI_RELEASE: /home_local/github-runner/testing_lib/baci_master/release/baci-release
+  # Some test require the 4C executable to check if the created input file
```
Comment (davidrudlstorfer) [inline .github/workflows/testing.yml:20]:
```suggestion
  # Some tests require the 4C executable to check if the created input file
```

File: meshpy/coupling.py [hunk 2]
```diff
@@ -42,7 +42,7 @@
 
 
 class Coupling(BoundaryConditionBase):
-    """Represents a coupling between geometry in BACI."""
+    """Represents a coupling between geometry in 4C."""
```
Comment (davidrudlstorfer) [inline meshpy/coupling.py:45]:
```suggestion
    """Represents a coupling between geometries in 4C."""
```

File: meshpy/simulation_manager/simulation_manager.py [hunk 3]
```diff
@@ -327,21 +327,21 @@ def add(self, other):
...
+        **kwargs,
     ):
         """
         Create the script to run all simulations contained in this manager.
 
         Args
         ----
-        baci_build_dir: str
-            Build directory for baci.
+        FOUR_C_BUILD_DIR: str
```
Comment (davidrudlstorfer) [inline meshpy/simulation_manager/simulation_manager.py:341]:
```suggestion
        four_c_build_dir: str
```

File: tests/utilities.py [hunk 4]
```diff
@@ -61,11 +61,11 @@ def skip_fail_test(self, message):
         self.skipTest(message)
 
 
-def get_baci_path():
-    """Look for and return a path to baci-release."""
+def get_four_c_path():
+    """Look for and return a path to 4C executable."""
```
Comment (davidrudlstorfer) [inline tests/utilities.py:65]:
```suggestion
    """Look for and return a path to the 4C executable."""
```

**Review by davidrudlstorfer — DISMISSED** [review]
Nice! Thanks for all necessary changes for the switch to 4C and the replacements with f-strings! 

I only left a few minor remarks, LGTM 🚀

File: meshpy/simulation_manager/simulation_manager.py (hunk 3 above)
Comment (isteinbrecher) [reply meshpy/simulation_manager/simulation_manager.py:341]: I prefer to have all function arguments in lower case.

**Comment by isteinbrecher** [comment]
@davidrudlstorfer thanks four the review!

File: meshpy/simulation_manager/simulation_manager.py (hunk 3 above)
Comment (isteinbrecher) [reply meshpy/simulation_manager/simulation_manager.py:341]: Oh, I missed that one, I have a fixup pull request comming up.

**Pull request merged**